infrastructure/permission/validPermissions.go

```diff
@@ -0,0 +1,94 @@
+package permission
+
+import "github.com/ystv/web-auth/permission/permissions"
+
+func GetValidPermissions(p permissions.Permissions) (m map[string]bool) {
+	m = make(map[string]bool)
+	if p.GetString() == permissions.MenuDisabled.GetString() {
+		m[p.GetString()] = true
+		return
+	}
```

Comment:
NITPICK: Why is this not part of the `switch`?

Reply:
By the way this works, this would make any permission above/more powerful would also have it, I have added it to the switch with a return